packages/arcgis-rest-portal/test/items/update.test.ts

```diff
@@ -215,6 +215,150 @@ describe("search", () => {
+      const fakeItem = {
+        id: "5bc",
+        owner: "dbouwman",
+        access: "private",
+        title: "My fake item",
+        description: "Updated description",
+        thumbnail: "thumbnail.png",
+        type: "Web Mapping Application",
+        typeKeywords: ["fake", "kwds"],
+        tags: ["fakey", "mcfakepants"]
+      };
+      const itemUrl =
+        "https://myorg.maps.arcgis.com/sharing/rest/content/users/dbouwman/items/5bc/update";
+
+      fetchMock.once(itemUrl, { success: true });
+
+      updateItem({
+        item: fakeItem,
+        params: {
+          thumbnailUrl:
+            "https://myorg.maps.arcgis.com/sharing/rest/content/items/5bc/info/thumbnail.png"
+        },
+        authentication: MOCK_USER_SESSION
+      })
+        .then(() => {
+          expect(fetchMock.called(itemUrl)).toBe(true);
+          const [, options] = fetchMock.lastCall(itemUrl)!;
+          const body = options.body as any;
+          let thumbnailValue: any;
+          if (body instanceof FormData) {
+            const entries = Array.from((body as any).entries()) as [
+              string,
+              any
+            ][];
+            const thumb = entries.find(([key]) => key === "thumbnailUrl");
+            thumbnailValue = thumb ? thumb[1] : undefined;
+          } else if (typeof body === "string") {
+            const params = new URLSearchParams(body);
+            thumbnailValue = params.get("thumbnailUrl");
+          } else if (body instanceof URLSearchParams) {
+            thumbnailValue = body.get("thumbnailUrl");
+          }
+          expect(thumbnailValue).toBeNull();
+          done();
+        })
+        .catch((e) => {
+          fail(e);
+        });
+    });
+
+    it("should update thumbnail when thumbnail is a Blob", (done) => {
+      const fakeItem = {
+        id: "5bc",
+        owner: "dbouwman",
+        access: "private",
+        title: "my fake item",
+        description: "old description",
+        type: "Web Mapping Application",
+        typeKeywords: ["fake", "kwds"],
+        tags: ["fakey", "mcfakepants"]
+      };
+      const itemUrl =
+        "https://myorg.maps.arcgis.com/sharing/rest/content/users/dbouwman/items/5bc/update";
+      const blobThumbnail = new Blob(["thumbnail data"], { type: "image/png" });
+
+      fetchMock.postOnce(itemUrl, { success: true });
+
+      updateItem({
+        item: fakeItem,
+        params: { thumbnail: blobThumbnail },
+        authentication: MOCK_USER_SESSION
+      })
+        .then(() => {
+          expect(fetchMock.called(itemUrl)).toBe(true);
+          const [, options] = fetchMock.lastCall(itemUrl)!;
+          const body = options.body as any;
+          let thumbnailIsBlob = false;
+          if (body instanceof FormData) {
+            const entries = Array.from((body as any).entries()) as [
+              string,
+              any
+            ][];
+            const thumbEntry = entries.find(([key]) => key === "thumbnail");
```

Comment:
```suggestion
            const thumbEntry = (body as any).get("thumbnail");      
```

`FormData` has a getter that you can use instead of this. https://developer.mozilla.org/en-US/docs/Web/API/FormData/get